File: encontrar.py
#Elaborar un algoritmo dada un numero limite=n de aleatorios entre 1 y 100, encontrar las coincidencias de x
import random
def generar(n):
    numeroran=[]
    for i in range(n):
        numeroran.append(random.randint(1,100))
    return numeroran
def buscar(x,n):
    a=generar(n)
    con=0
    for i in a: con+=(i==x)
    return f"el numero de coincidencias para {x} en el conjunto {a} es {con}"
n=int(input())
x=int(input())
print(buscar(x,n))

File: encontrar.ts
import readline from "node:readline/promises";
import {stdin as input, stdout as output} from "node:process";
const rl=readline.createInterface({input,output});
function generar(limite:number){
    let numeroran=new Array();
    for(let i=0;i<limite;i++){
        numeroran.push(Math.random()*100);
    }
    return numeroran;
}
function buscar(limite:number,x:number){
    let a=generar(limite);
    let con=0;
    a.forEach(element => {
        con+=Number(element==x);
    });
    return {
        "mensaje":`El numero de coincidencias de ${x} es ${con}`,
        "resultado":a
    }
}